### DISCUSSION TITLE
chipset fan

### DISCUSSION THREAD

**Discussion by ironpink227** [Rem0o/FanControl.Releases#476] [Q&A]
Hi, in case of x570 (chipset with fan) is there a chance to get chipset temp?
tx
raffaele

**Comment by Gozzima** [comment]
On my Asus Prime X570-Pro, there are 6 temp probes which are seen by FanControl as soon as "Controller (Various Third Parties)" is checked in Sensor settings. I suspect the hotest one is the chipset (Temperature #2 or #4 for me). Chipset fan RPM is not read by FanControl for me.

**Reply by ironpink227** [reply]
try to check the temperature on hwinfo. I'm same issue and I can't set it properly (so I don't use it)
raffaele

![scr_01](https://user-images.githubusercontent.com/97922099/150297133-6fdf88d7-722a-48f5-998c-099fc834d8b8.jpg)
![scr_02](https://user-images.githubusercontent.com/97922099/150297137-6783e251-50ad-4b7f-be57-277ae50cb0d0.jpg)
.

**Comment by Globespy** [comment]
I have the same issue - Fan Control picks up the fan speed of the chipset fan correctly, however does not seem to have the correct temperature (not avail in the "various third party" options)
![image](https://user-images.githubusercontent.com/8172493/186226528-a876f852-3b29-4b26-9a22-24986b73632f.png)

**Comment by Rem0o** [comment]
Feel free to use https://github.com/Rem0o/FanControl.HWInfo if you got it running anyway.

**Reply by Globespy** [reply]
Thank you, I am sorry I don't quite understand the instructions?
I downloaded the latest (4 files including the dll you mention), but don't understand how to 'compile'?
I have enabled the other things in HWInfo64 and tried copying the 'FanControl.HWInfo.dll' into the Fan Control plugins folder, but it's not recognizing the chipset value I need to create a curve in Fan Control.
Help appreciated.

**Reply by Rem0o** [reply]
Right click property "FanControl.HWInfo.dll", and unlock it.  Then relaunch FC.

**Reply by Globespy** [reply]
Thanks!
Unfortunately, it seems that the motherboard (Aorus X570 Master) is overriding any attempts to control this fan, even with the new 
Auto (BETA) curve.

**Comment by Globespy** [comment]
I also get this error a lot when starting (I open HWInfo64 before Fan Control).
Do you have any way for Fan Control to open HWInfo64 'silently' on launch since it requires it to be open to use other sensors not picked up by Fan Control?
Seems like Fan Control cannot understand the Chipset controller 'ITE IT8688E'?
Thanks
![image](https://user-images.githubusercontent.com/8172493/187537902-e57c3ac1-07d2-4a4c-9086-f4bc0014279e.png)

**Reply by Rem0o** [reply]
Are you sure HWInfo is opened when that error occurs? That just means the sensor was not detected from HWInfo.

**Reply by Globespy** [reply]
Hi Remi,
Yes, this message would show sometimes even when HWInfo64 was open when Fan Control launched.
I could not get Fan Control to accurately or reliably control the PCH Motherboard fan using the HWInfo64 sensor from the plugin, mostly caused when the fan would spin down to zero RPM (silent profile in BIOS) - this would result in the fan speed no longer being seen in Fan Control. Admittedly, I think this is more a problem caused by HWInfo64 being unable to reliably relocate the PCH fan RPM after it's spun down to 0% under the 'silent' BIOS option - this is documented in HWInfo64.

I decided to simply choose the 'balanced' PCH option in BIOS, then using the significantly less accurate chipset temp (based on diodes 'near' the chipset instead of from within the chipset itself as reported by HWInfo64) to muddle together a simple fan curve that at least means I can see the PCH fan speed - I based the speed/temp curve on how the 'diode' PCH temp sensor changes with the real internal chipset temp, so it works somewhat, even though I can no longer use the 'silent' BIOS option where the fan will spin down to zero - seems the lowest % before the PCH fan stops reporting RPM in Fan Control using this method is 12%, or about 1050RPM. 
Not ideal, but at least I can always see the fan now!

Thanks again for your wonderful program!
